c++/build.sh

```diff
@@ -1,3 +1,4 @@
 #!/usr/bin/env sh
 clang++ -std=c++11 -Wall -Wextra -Werror -g ./src/linked-list.cc -o out/linked-list.bin
-clang++ -std=c++11 -Wall -Wextra -Werror -g ./src/binary-tree.cc -o out/binary-tree.bin
\ No newline at end of file
+clang++ -std=c++11 -Wall -Wextra -Werror -g ./src/binary-tree.cc -o out/binary-tree.bin
+clang++ -std=c++11 -Wall -Wextra -Werror -g ./src/vector.cc -o out/vector.bin
```

Comment:
Here's a [simple guide](http://mrbook.org/blog/tutorials/make/) on the basics of makefiles if you prefer to go that route.